### DISCUSSION TITLE
umi + styled-components，如何给div传递css prop

### DISCUSSION THREAD

**Discussion by JiangChengen** [umijs/umi#11888] [Q&A]
环境：
![image](https://github.com/umijs/umi/assets/35300738/f052c60e-dd32-4a2f-9735-d93f1925f94f)
.umirc配置：
![image](https://github.com/umijs/umi/assets/35300738/e17a39ee-5080-40df-bb97-ba98d08fbdca)
翻阅`styled-components`文档，文档中提到可以通过如下方式给`HTML`标签增加css属性:
![image](https://github.com/umijs/umi/assets/35300738/c17044fe-b3bc-4756-a7b2-99742fc8335c)
文档地址：[css-prop](https://styled-components.com/docs/api#css-prop)
按照文档：
```jsx
<div
  css={`
    color: #fff;
  `}
></div>
```
报错：
![image](https://github.com/umijs/umi/assets/35300738/7bf02d9d-8b25-47fa-a036-33cfafbebbd5)
按照`styled-components`文档的意思是要给项目增加babel插件：`babel-plugin-styled-components`，
又参考UMI文档，给umirc增加`styledComponents.babelPlugin`就是已经添加了这个插件。

请问各位大佬你们遇见过么，还是说我的配置问题

**Comment by fz6m** [comment]
如果用了 umi 再带的 styled components 插件的话，需要从 umi 导入 SC 的方法：

```ts
import { styled, css } from 'umi'
```

如有问题，需要给一个最小复现。

**Reply by JiangChengen** [reply]
不是的，styled-component官方文档说的是可以给指定的html标签传递css属性，然后通过它的babel插件读取到这样的用法就可以转换成一个sc组件，而不是说通过从umi或者sc里面引入css函数来创建样式。

**Reply by fz6m** [reply]
你说得对，这个问题预计在 #11898 修复。

**Reply by JiangChengen** [reply]
好的 谢谢
